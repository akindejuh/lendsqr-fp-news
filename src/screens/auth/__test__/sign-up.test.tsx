import React from 'react';
import 'react-native';
import SignUpScreen from '../sign-up';
import { NavigationContainer } from '@react-navigation/native';
import { SafeAreaProvider } from 'react-native-safe-area-context';
import renderer from 'react-test-renderer';
import { test, expect } from '@jest/globals';

test('Sign Up Snapshot', () => {
  const snap = renderer
    .create(
      <SafeAreaProvider>
        <NavigationContainer>
          <SignUpScreen />
        </NavigationContainer>
      </SafeAreaProvider>,
    )
    .toJSON();

  expect(snap).toMatchSnapshot();
});
