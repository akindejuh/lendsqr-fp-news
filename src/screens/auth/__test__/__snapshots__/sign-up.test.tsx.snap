// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign Up Snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
